protoc-gen-swagger/options/openapiv2.proto

```diff
@@ -94,13 +148,20 @@ message Response {
 //
 // See: https://github.com/OAI/OpenAPI-Specification/blob/3.0.0/versions/2.0.md#infoObject
 //
-// TODO(ivucica): document fields
 message Info {
+  // Required. The title of the application.
   string title = 1;
+  // A short description of the application. GFM syntax can be used for rich
+  // text representation.
   string description = 2;
+  // The Terms of Service for the API.
   string terms_of_service = 3;
+  // The contact information for the exposed API.
   Contact contact = 4;
+  // The license information for the exposed API.
   License license = 5;
+  // Required Provides the version of the application API (not to be confused
```

Comment:
```suggestion
  // Required. Provides the version of the application API (not to be confused
```

Reply:
Agreed, I was a little unsure about adding this as obviously there is a difference between what the swagger spec defines as required and what is required for protoc-gen-swagger. I'll remove the required statement. I guess we could document the defaults at some point.

Reply:
It looks like there are a number of other descriptions in the proto definition which look like they have been copied from the open api spec verbatim with 'required'. Do you think we should also remove this?

Reply:
Haha, yeah, just left a comment to this effect.

Reply:
I've not removed all the other instances of required.